### PR TITLE
Add useNewUrlParser options to GridFSBucketAdapter

### PR DESCRIPTION
**Pull request by dplewis**
Closes: https://github.com/parse-community/parse-server/issues/5525

### CODE REVIEW

File: src/Adapters/Files/GridFSBucketAdapter.js [hunk 1]
```diff
@@ -14,17 +14,21 @@ import defaults from '../../defaults';
...
   _databaseURI: string;
   _connectionPromise: Promise<Db>;
+  _mongoOptions: Object;
 
-  constructor(mongoDatabaseURI = defaults.DefaultMongoURI) {
+  constructor(mongoDatabaseURI = defaults.DefaultMongoURI, mongoOptions = {}) {
     super();
     this._databaseURI = mongoDatabaseURI;
+    this._mongoOptions = mongoOptions;
+    this._mongoOptions.useNewUrlParser = true;
```
Comment (acinader) [inline src/Adapters/Files/GridFSBucketAdapter.js]: does it not work with the old parser?

if it doesn't, this looks fine.

if the old parser does work, then something like:

```
const defaultMongoOptions = { useNewUrlParser: true };
this._mongoOptions = Object.assign(defaultMongoOptions, mongoOptions);
```

Comment (dplewis) [reply src/Adapters/Files/GridFSBucketAdapter.js]: I don't know as I use AWS S3. https://github.com/parse-community/parse-server/issues/5525 posted that this change fixed it for them. Should I make your requested change?

Comment (acinader) [reply src/Adapters/Files/GridFSBucketAdapter.js]: well, not without testing it, but I think so, yes. I don't see any reason not to allow a user to override the default.

Comment (dplewis) [reply src/Adapters/Files/GridFSBucketAdapter.js]: Oh I see, this will allow for both.

**Pull request merged**